cmd/root_test.go

```diff
@@ -733,7 +728,7 @@ func TestBustCacheOnFormatterChange(t *testing.T) {
 	})
 }
 
-func TestGitWorktree(t *testing.T) {
```

Comment:
Did you mean to change the test name? Totally fine if you did, I just know this PR evolved a bit before reaching it's final form.

Reply:
I did